wrappers/ortools.rb

```diff
@@ -551,6 +551,7 @@ def parse_output(vrp, _services, points, _matrix_indices, _cost, _iterations, ou
                   previous_matrix_index = points[vehicle.start_point.id].matrix_index
                   current_activity = {
                     point_id: vehicle.start_point.id,
+                    type: 'store',
```

Comment:
That's the first time, the word 'store' is used to define the depots. Is there a particular reason to change the naming ?
depots are mentioned indirectly multiple times within the documentation and the API code